### ISSUE TITLE
1st iteration feedback

### ISSUE THREAD

**Issue by huberflores** [MuradPats/softproj#88]
Please find below the feedback from your work done in the 1st iteration.

Wiki
Homepage (Vision)
- All the other pages should be linked from this main page, e.g., functional requirements, non-functional requirements, meetings, etc.

Functional requirements.
- The set of requirements provided are good. However, some more details can be added during the development of the project. Some of them should be extended to include more specific information, for instance, what are the sensor threshold to determine that the phone is being shaken, or what is the threshold that differentiates normal from hard shaking.
- Details also include the data to be used, e.g., user can communicate with bank representative via text messages, what the message should contain? or does the application creates a template from the message to be sent?

Non-Functional requirements
- Try to locate functional requirements into a well defined category, for instance, 1. Usability, 1.1. Notifications after login should appear no sooner than 10 sec, 1.2. App must be fully usable on at least 90% of Android devices. 2. Fault tolerance, 2.1....., 2.2.... 3. Reliability, 3.1...., 3.2.....
- It will be a good idea also to mention what is the alternative functionality if the application fails for all the requirements, for instance, what happens if the notification does not reach the phone in the 10 seconds umbral?
- Some requirements are not very clear, more specific details are missing. Reading the critic section of this link may help to understand the issue (http://users.csc.calpoly.edu/~jdalbey/SWE/QA/nonfunctionalExamples.html) 

Project Plan
- I just requested access as it is located in Google docs. I am expecting to see how the project will be developed during each iteration of the project. Basically, the plan is based on the requirements that were described.

Team structure.
- The roles are brief and clear. 

Team communication with the customer
- During the project, it would be good to schedule regular meetings, which are fixed in a schedule. 

Work process
- This section is fine, but it may benefit from some polishing and restructuring. For instance, you can start by saying what kind of methodology you will follow, e.g., SPRING methodology. Then, you can mention the technologies that you will use to manager your project, e.g., Trello and Toggl. Finally, you can describe the iterative process you will follow, this includes, integrating feedback from customer, releasing a version of the the application, verification of the feedback integrated in the latest release and so on.

Wiki
- Meeting page is missing. This page contains what have you talked with your customer and mentor. Moreover, it contains the goals/objectives/changes/tasks which will be improved or added.
- Meetings with the customer and mentor also should be linked from the homepage by a "meetings" page.

Issue tracker
- This is well done.

VCS.
- Issues and sub-issues should be linked (when applicable). Each commit should be associated their respective issue. Finally, remember to close each issue with the last token commit as seen in the meeting.

Transparency. 
- Not graded in this iteration, but it has to be considered from now. This point means that all the requirements, wiki, issue tracker, etc. are well integrated. In short, there should be consistency in the project, e.g., the requirements of the wiki must match the issues of the issue tracker.

Iterative tuning
- This issue can be used for tuning 1st iteration deliverable. Thus, after you applied changes, it may be possible to extend the discussion.

**Comment by tarstarkvara** [comment]
Non-functional requirements categorised and corrected, meetings page created

**Comment by huberflores** [comment]
Unlimited - http://pencil.evolus.vn/
Although, the look and feel of the tools with free trial period is better - http://c2.com/cgi/wiki?GuiPrototypingTools

I think that the trial period of some tools is enough for your project. For instance, this one (https://www.ninjamock.com/account/pricing) provides a maximum of 3 projects per user in the free plan.
